lib/redis-stat/elasticsearch.rb

```diff
@@ -43,7 +44,8 @@ def output info
         :body  => entries.merge({
           :@timestamp => format_time(time),
           :host       => host,
-          :at         => time.to_f
+          :at         => time.to_f,
+          :tags       => @tags
```

Comment:
Is it okay that we leave `:tags` when the value is empty? i.e. Is nil entry automatically ignored? If not, we should either 1. filter out the `:tags` from the hash or 2. set it only when `@tags` is not nil.

- `.select { |k, v| v }`
    - This one is simpler, but can have different result on Ruby 1.8
- `{ ... without :tags here ... }.tap { |hash| hash[:tags] = @tags if @tags }`